src/app/services/actions/ocps-action-creators.ts

```diff
@@ -0,0 +1,89 @@
+import { authenticationWrapper } from '../../../modules/authentication';
+import { IAction } from '../../../types/action';
+import { IRequestOptions } from '../../../types/request';
+import {
+  GET_POLICY_ERROR,
+  GET_POLICY_PENDING,
+  GET_POLICY_SUCCESS
+} from '../redux-constants';
+
+interface IPolicyValues {
+  email: number,
+  screenshot: number,
+  feedback: number
+}
+
+export function getPoliciesSuccess(response: object): IAction {
+  return {
+    type: GET_POLICY_SUCCESS,
+    response
+  };
+}
+
+export function getPoliciesError(response: object): IAction {
+  return {
+    type: GET_POLICY_ERROR,
+    response
+  };
+}
+
+export function getPoliciesPending(): any {
+  return {
+    type: GET_POLICY_PENDING
+  };
+}
+
+
+// TODO: Test this function
+export function getPolicies(): Function {
+  return async (dispatch: Function) => {
+    const policyUrl = 'https://sip.clients.config.office.net/user/v1.0/web/policies';
+    const token = await authenticationWrapper.getOcpsToken();
```

Comment:
This function could have a default return type for when getting the OCPS token fails. This should help invalidate the token so that it throws the function into dispatching the getPoliciesError function.

Alternatively it can throw an error and be wrapped in a try catch that immediately dispatches the getPoliciesError  function